scr/sysbox

```diff
@@ -6,9 +6,13 @@
 # Note: normally Sysbox is installed via a distro-specific package
 # which sets up the Sysbox systemd units. This script is meant as a
 # quick alternative to the installer to help launch Sysbox manually
-# (e.g., for testing).
+# (e.g. for development & testing).
```

Comment:
Now that we've added a cmd line param for this script, let's add a comment regarding the expected invocation:

./sysbox <testing-mode>

Reply:
Right. Done.